libs/libsndfile/Makefile

```diff
@@ -27,8 +27,7 @@ define Package/libsndfile
   SECTION:=libs
   CATEGORY:=Libraries
   TITLE:=Library for reading/writing audio files
-  URL:=http://www.mega-nerd.com/libsndfile/
-  MAINTAINER:=Peter Wagner <tripolar@gmx.at>
+  URL:=https://libsndfile.github.io/libsndfile/
```

Comment:
Need this:
  DEPENDS:=+lame-lib +libmpg123

Reply:
Makes sense, done